fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/autobrakes.rs

```diff
@@ -229,7 +229,8 @@ impl A380AutobrakeController {
             active_id: context.get_identifier("AUTOBRAKES_ACTIVE".to_owned()),
             rto_mode_armed_id: context.get_identifier("AUTOBRAKES_RTO_ARMED".to_owned()),
 
-            external_disarm_event_id: context.get_identifier("AUTOBRAKE_DISARM".to_owned()),
+            external_disarm_event_id: context
```

Comment:
```suggestion
            external_deactivation_event_id: context
```
and whereevery else this occurs :)